src/lib/workspace/Variable.tsx

```diff
@@ -84,8 +84,7 @@ export function VariableDetails(props: Props) {
               {showMore && providerLabelLeftover}
               &nbsp;
               <HelpIcon>
-                The name of this variable in the data files that were integrated
-                into ClinEpiDB
+                The name of this variable in the orignal data files
```

Comment:
@dmfalke That makes sense. @danicahelb or @SheenaTomko, any feedback on the above change?

Reply:
I'm ok with it. @danicahelb? I know the files we integrate aren't always "original". If you want something like "The name of this variable in the data file it was integrated from" I would be ok with that too

Reply:
@danicahelb To be clear, which verbiage did you approve? Mine above or Sheena's below mine?

Reply:
I believe in the meeting she said the suggestion I put forth above. I'll merge, but feel free to correct me, @danicahelb!